tests/net/ieee802154/l2/src/ieee802154_shell_test.c

```diff
@@ -201,7 +209,7 @@ ZTEST(ieee802154_l2_shell, test_active_scan)
 		0x00, 0xc0, /* Superframe Specification: PAN coordinator + association permitted */
 		0x00, /* GTS */
 		0x00, /* Pending Addresses */
-		0x00, 0x00 /* Payload */
+		EXPECTED_PAYLOAD_DATA /* Payload */
```

Comment:
```suggestion
		EXPECTED_PAYLOAD_DATA
```
Nice solution, very readable. Like it.